packages/ra-ui-materialui/src/input/ArrayInput/ArrayInput.stories.tsx

```diff
@@ -167,6 +167,47 @@ export const Scalar = () => (
     </Admin>
 );
 
+export const ScalarI18n = () => (
+    <Admin
+        dataProvider={dataProvider}
+        history={history}
+        i18nProvider={testI18nProvider({
+            messages: {
+                resources: {
+                    books: {
+                        fields: {
+                            tags: 'Some tags',
+                        },
+                    },
+                },
+            },
+        })}
+    >
+        <Resource
+            name="books"
+            edit={() => (
+                <Edit
+                    mutationMode="pessimistic"
+                    mutationOptions={{
+                        onSuccess: data => {
+                            console.log(data);
+                        },
+                    }}
+                >
+                    <SimpleForm>
+                        <TextInput source="title" />
+                        <ArrayInput source="tags" fullWidth>
+                            <SimpleFormIterator disableReordering>
+                                <TextInput source="" helperText={false} />
```

Comment:
This story is great for testing the corner case where we have no `label`, but it should demonstrate the recommended way to do a scalar array input, and have a `label` by default (besides, otherwise, it raises the newly added warning in `useInput`)